### DISCUSSION TITLE
Keycloak 502 Bad Gateway Issue Upon Angular Application Refresh or Manual URL Change After Authentication

### DISCUSSION THREAD

**Discussion by MukeshAnandan** [keycloak/keycloak#26679] [Q&A]
I'm encountering difficulties while integrating Keycloak authentication into my Angular application using server configuration by creating respective realms and clients.

Issue I'm facing:

Upon launching the Angular application, it redirects to the keycloak authentication page. After signing in and navigating to the application page, the problem arises when I refresh the page or manually change the application URL. This leads to a keycloak 502 bad gateway issue. However, if I perform a hard refresh on the keycloak authentication page before signing in, everything works fine (both refreshing the page and manually changing the URL).

Please share any insights you may have on this issue or suggestions on how to overcome it.

**Comment by codegastudio** [comment]
Exactly same problem !
The only way to be able to retrieve access is to clean navigator history. Then we get back access to login page.
Our app is deployed in a cluster behind an nginx server and ingress controller. 
We have try to configure nginx through ingress annotation like described everywhere with the following key/value but nothing change. Our nginx server is up to date. 
```
nginx.ingress.kubernetes.io/proxy-buffer-size: "128k"
```

**Comment by vishnudutv** [comment]
Were you able to solve the issue @MukeshAnandan ?
